server/src/main/kotlin/fi/oph/kitu/auth/WebSecurityConfig.kt

```diff
@@ -41,10 +69,11 @@ class WebSecurityConfig {
                     "/db-scheduler-api/**",
                 )
             }
+            securityMatcher({ request ->
+                !isOauth2Request(request)
+            })
             logout {
                 logoutSuccessUrl = casConfig.getCasLogoutUrl()
-                logoutUrl = "/logout"
-                logoutRequestMatcher = AntPathRequestMatcher("/logout", "GET")
```

Comment:
This line has multiple issues:

1. `AntPathRequestMatcher` is deprecated and will be removed in Spring Boot 4.
2. It's arguably bad to allow logging out via a `GET` method, since it's not an idempotent operation and so can be cached. The default is to only allow `POST` methods.
3. The `/logout` path is the default value anyway.